src/agent/debuglet.ts

```diff
@@ -535,7 +536,7 @@ export class Debuglet extends EventEmitter {
       that.logger.info('Fetching breakpoints');
       // TODO: Address the case when `that.debuggee` is `null`.
       that.controller.listBreakpoints(
-          (that.debuggee as Debuggee), function(err: Error, response, body) {
+          (that.debuggee as Debuggee), function(err: Error|null, response, body) {
```

Comment:
question: could the err type annotation be omitted

Reply:
Yes. Removed.